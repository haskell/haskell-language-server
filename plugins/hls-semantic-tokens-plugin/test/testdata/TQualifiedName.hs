module TQualifiedName where

import qualified Data.List


a = Prelude.undefined
b = 1 `Data.List.elem` [1, 2]
c = (Prelude.+) 1 1
d = (+) 1 1
