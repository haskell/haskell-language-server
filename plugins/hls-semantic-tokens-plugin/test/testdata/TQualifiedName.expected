4:1-2 TVariable "a"
4:5-13 TModuleName "Prelude."
4:13-22 TVariable "undefined"
5:1-2 TVariable "b"
5:8-16 TModuleName "Prelude."
5:16-20 TClassMethod "elem"
6:1-2 TVariable "c"
6:6-14 TModuleName "Prelude."
6:14-15 TClassMethod "+"
